styles/_header.scss

```diff
@@ -1,17 +1,47 @@
 /***** Header *****/
-$header-height: 71px;
+
 
 .header {
-  @include max-width-container;
-  position: relative;
-  align-items: center;
-  display: flex;
-  height: $header-height;
-  justify-content: space-between;
+  background-color: $white;
+  margin: 0 auto;
+  max-width: $max-width-page-container;
+
+  &__inner {
+    @include max-width-container;
+    position: relative;
+    align-items: center;
+    display: flex;
+    @include fluid-value( height, $min-width, $max-width, 100px, 138px);
+    justify-content: space-between;
+  }
 }
 
-.logo img {
-  max-height: 37px;
+.logo {
+
+  img {
+    @include fluid-value( max-height, $min-width, $max-width, 25px, 35px);
+    @include tablet { 
```

Comment:
is there a reason you went with `@include tablet` over something like `include breakpoint()`